### PR TITLE
pj-rehearse: be better at logging

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

The semi-machine-read-able logs are hard to parse but this at least fixes areas where we'd not output anything at all when choosing a job.

### CODE REVIEW

File: cmd/pj-rehearse/main.go [hunk 1]
```diff
@@ -132,9 +133,7 @@ func rehearseMain() error {
 		jobSpec.Refs.Pulls[0].Number = int(time.Now().Unix())
 	}
 	org, repo, prNumber := jobSpec.Refs.Org, jobSpec.Refs.Repo, jobSpec.Refs.Pulls[0].Number
-
-	logger = logrus.WithField(prowgithub.PrLogField, prNumber)
-	logger.Info("Rehearsing Prow jobs for a configuration PR")
+	logger.Infof("Rehearsing Prow jobs for configuration PR %s/%s#%d", org, repo, prNumber)
```
Comment (alvaroaleman) [inline cmd/pj-rehearse/main.go:136]: Can we please make these fields and create a logger that we continue using?

Comment (stevekuznetsov) [reply cmd/pj-rehearse/main.go:136]: I really oppose that. We're not reading this with a machine and my experience reading a log today strongly made me rethink this approach ... I don't need to see the PR number on every line of the log, we only ever operate on one PR at a time.
